protocol/auditlog.go

```diff
@@ -100,6 +116,7 @@ func (l ConiksAuditLog) Insert(addr string, signKey sign.PublicKey,
 	// create the new directory history
 	h = newDirectoryHistory(addr, signKey, snaps[0])
 
+	// FIXME: remove this check --> caller calls Audit() before this function
```

Comment:
Since we will pass Response message as param, should we assume that the callers will call Audit() before this function? Or is it better to embed the Audit() call in this function?